docs/deploy-from-precompiled-binaries-windows.md

```diff
@@ -0,0 +1,105 @@
+---
+date: '2019-10-16'
+title: 'Deploy Platform V3 from precompiled binaries on Windows'
+layout: docs
+---
+## Summary
+
+Use this guide to <a class="crosslink" href="https://virtocommerce.com/ecommerce-hosting" target="_blank">deploy</a> and configure precompiled Virto Commerce Platform V3.
+
+### Prerequisites
+
+[Prerequisites for .NET Core on Windows](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites)
+
+[.NET Core SDK](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-windows-x64-installer)
+
+[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/sql-server-2017-editions)
+
+## Downloading the precomplied binaries
+
+* Navigate to the <a href="https://github.com/VirtoCommerce/vc-platform/releases">Releases section of Virto Commerce Platform in GitHub.</a>
+
+* You will find **VirtoCommerce.Platform.3.x.x.zip** file. In this file the site has already been built and can be run without additional compilation. It does not includes all the source code.
+
+* Unpack this zip to a local directory **C:\vc-platform-3**. After that you will have the directory with Platform precompiled files.
+
+## Setup
+
+### Configure application strings
+
+* Open the **appsettings.json** file in a text editor.
+* In the **ConnectionStrings** section change **VirtoCommerce** node (provided user should have permission to create new database):
+
+```json
+    "ConnectionStrings": {
+        "VirtoCommerce" : "Data Source={SQL Server URL};Initial Catalog={Database name};Persist Security Info=True;User ID={User name};Password={User password};MultipleActiveResultSets=True;Connect Timeout=30"
+    },
+
+```
+
+* In the **Assets** section set public url for assets `Assets:FileSystem:PublicUrl` with url of your application, this step is needed in order for display images
+
+```json
+"Assets": {
+        "Provider": "FileSystem",
+        "FileSystem": {
+            "RootPath": "~/assets",
+            "PublicUrl": "https://localhost:5001/assets/" <-- Set your platform application url with port localhost:5001
+        },
+    },
+```
+
+### Running the Platform by CLI "dotnet"
+
+* Install and trust HTTPS certificate
+
+To trust the .NET Core SDK HTTPS development certificate perform run:
```

Comment:
Run to trust the .NET Core SDK HTTPS development certificate:

Reply:
Fixed